src/core/blocks/core.cpp

```diff
@@ -1006,8 +1006,43 @@ struct Assoc : public VariableBase {
   ExposedInfo _requiredInfo{};
   Type _tableTypes{};
 
+  static inline Parameters params{
+      {"Name",
+       CBCCSTR("The name of the sequence or table to be updated."),
+       {CoreInfo::StringOrAnyVar}},
+      {"Key",
+       CBCCSTR("Table key for the value that is to be updated. Parameter "
+               "applicable if target is table."),
+       {CoreInfo::StringStringVarOrNone}},
+      {"Global",
+       CBCCSTR("If the variable is or should be available to all the chains in "
+               "the same node. The default value (false) makes the variable "
+               "local to the "
+               "chain."),
+       {CoreInfo::BoolType}}};
+  static CBParametersInfo parameters() { return params; }
+
+  static CBOptionalString help() {
+    return CBCCSTR("Updates a sequence (array) or a table (associative array/ "
+                   "dictionary) on the basis of an input sequence.");
+  }
+
   static CBTypesInfo inputTypes() { return CoreInfo::AnySeqType; }
+  static CBOptionalString inputHelp() {
+    return CBCCSTR(
+        "Input sequence that identifies element to be updated and "
+        "its new value. The sequence must contain even-number of elements "
+        "with the 1st element in each pair denoting the target index to update "
+        "and the 2nd element holding the update value for the corresponding "
+        "index.");
```

Comment:
Here also it looks like the split is incorrect.

Unfortunately, `ninja format` doesn't seem to rebuild the whole string and only looks at each line length separately. Therefore, you do have to put it back in one line for the split to occur again correctly.

Reply:
I have moved the last word 'index' into the previous line.